handlers/tribes_test.go

```diff
@@ -648,33 +648,38 @@ func TestGetAllTribes(t *testing.T) {
 }
 
 func TestGetTotalTribes(t *testing.T) {
-	mockDb := mocks.NewDatabase(t)
-	tHandler := NewTribeHandler(mockDb)
+	teardownSuite := SetupSuite(t)
+	defer teardownSuite(t)
+	tHandler := NewTribeHandler(db.TestDB)
 	t.Run("should return the total number of tribes", func(t *testing.T) {
 		rr := httptest.NewRecorder()
 		handler := http.HandlerFunc(tHandler.GetTotalribes)
 
-		expectedTribes := []db.Tribe{
-			{UUID: "uuid", Name: "Tribe1"},
-			{UUID: "uuid", Name: "Tribe2"},
-			{UUID: "uuid", Name: "Tribe3"},
-		}
+		db.TestDB.DeleteTribe()
 
-		expectedTribesCount := int64(len(expectedTribes))
+		expectedTribesCount := int64(1)
```

Comment:
The expectedTribesCount should come from the DB as stated in the Issue, and not static, this will lead to future errors

Reply:
@elraphty it is coming from db 
![image](https://github.com/stakwork/sphinx-tribes/assets/98140673/2ea2ce13-7cef-47c5-8f5b-bf0a4cb8d395)

Reply:
@elraphty As i am creating the one tribe there for I expect that ( 1 EQUAL TO tribeCount)

Reply:
@elraphty 
![image](https://github.com/stakwork/sphinx-tribes/assets/98140673/bdc5ccc9-27da-402e-84e6-83b7a9a4ddc5)

Reply:
> @elraphty As i am creating the one tribe there for I expect that ( 1 EQUAL TO tribeCount)
Then let's remove the expectedTribesCount variable.

Reply:
> @elraphty As i am creating the one tribe there for I expect that ( 1 EQUAL TO tribeCount)

This may fail if other user's don't delete their own tribes